vpath/pjtester.yaml

```diff
@@ -0,0 +1,11 @@
+pjNames:
+  - pjName: "telemetry-operator"
+    pjPath: "test-infra/prow/jobs/kyma/components/telemetry-operator"
+    report: true
+  - pjName: "orphaned-disks-cleaner"
+  - pjName: "post-main-kyma-gke-integration"
+prConfigs:
```

Comment:
I don't think you need that no-op kyma PR, it is optional and only needed if you want to also test against certain changes on kyma, for example fast-integration test changes, new tests etc. If you omit this it will just use latest main of kyma